go.mod

```diff
@@ -1,11 +1,12 @@
 module github.com/openshift-kni/oran-o2ims
 
-go 1.22.5
```

Comment:
I don't think we're ready to move the operator to go 1.23 yet. This should be a separate activity. This PR should remain go 1.22.